CONTRIBUTING.rst

```diff
@@ -36,6 +36,7 @@ First step is setting up development environment and installing all the required
     $ source venv/bin/activate
 
     $ make setup
+    $ make package
```

Comment:
This should belong in the Makefile, just add this command [here](https://github.com/vacanza/python-holidays/blob/58cd972d6a5f96ab170241945504ecf878a96925/Makefile#L52)

Reply:
OK, I have followed your advice.